cmd/bnscli/cmd_distribution.go

```diff
@@ -16,14 +16,14 @@ import (
 func cmdResetRevenue(input io.Reader, output io.Writer, args []string) error {
 	fl := flag.NewFlagSet("", flag.ExitOnError)
 	fl.Usage = func() {
-		fmt.Fprintln(flag.CommandLine.Output(), `
+		_, _ = fmt.Fprintln(flag.CommandLine.Output(), `
 Create a transaction for reseting a revenue stream with a new configuration.
 		`)
 		fl.PrintDefaults()
 	}
 	revenueFl := flHex(fl, "revenue", "", "A hex encoded ID of a revenue that is to be altered.")
 	recipientsFl := fl.String("recipients", "", "A path to a CSV file with recipients configuration. File should be a list of pairs (address, weight).")
-	fl.Parse(args)
+	_ = fl.Parse(args)
```

Comment:
I think this should actually return the error and early abort.

Nice to see this diff (thanks to linter) to catch that unhandled error

Reply:
Yeah, same as below it's caught as panic. But I think you are right